src/iuse.cpp

```diff
@@ -2396,10 +2390,24 @@ int iuse::crowbar( player *p, item *it, bool, const tripoint &pos )
     }
 
     /** @EFFECT_STR speeds up crowbar prying attempts */
-
-    p->mod_moves( -( std::max( 20, ( difficulty * 20 ) - ( ( p->str_cur ) * 5 ) ) ) );
+    p->mod_moves( -std::max( 20, difficulty * 20 - p->str_cur * 5 ) );
     /** @EFFECT_STR increases chance of crowbar prying success */
 
+    // Doors need PRY 2 which is on a crowbar, crates need PRY 1 which is on a crowbar
+    // & a claw hammer. 
+    // The iexamine function for crate supplies a hammer object.
+    // So this stops the player (A)ctivating a Hammer with a Crowbar in their backpack
+    // then managing to open a door.
+    const int pry_level = it->get_quality( quality_id( "PRY" ) );
+
+    if( pry_level < pry_quality ) {
+        p->add_msg_if_player( _( "You can't get sufficient leverage to open that with your current equipment" ) );
```

Comment:
```suggestion
        p->add_msg_if_player( _( "You can't get sufficient leverage to open that with your %s" ), it->tname() );
```

Reply:
Ooh, fair shout; didn't think about specifically naming it.